config/environments/development.rb

```diff
@@ -18,7 +18,7 @@
     config.action_controller.perform_caching = true
     # config.action_controller.enable_fragment_cache_logging = true
 
-    config.cache_store = :redis_cache_store, {url: ENV['REDIS_URL']}
+    config.cache_store = :redis_cache_store, url: ENV['REDIS_CACHE_URL']
```

Comment:
Probably we need to set this somewhere for development? Docker-compose or something?

Reply:
Ooh, yeah – we have

```yaml
REDIS_URL: redis://redis
```
in there

Reply:
Yeah, dev doesn't use caching *by default* but we probably still want something it works with for testing caching, and similar.